package.json

```diff
@@ -25,8 +25,11 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@onflow/fcl": "^0.0.78",
+    "@onflow/util-encode-key": "^0.0.2",
     "@tatumio/bitcoincashjs2-lib": "^4.1.2",
     "axios": "^0.24.0",
+    "axios-retry": "^3.2.4",
```

Comment:
why do we need axios retry?

Reply:
You are right, no need for this. I will remove it.